Documentation/cmdref/cilium-agent.md

```diff
@@ -131,7 +131,7 @@ cilium-agent [flags]
       --endpoint-status strings                              Enable additional CiliumEndpoint status features (controllers,health,log,policy,state)
       --envoy-log string                                     Path to a separate Envoy log file, if any
       --exclude-local-address strings                        Exclude CIDR from being recognized as local address
-      --fixed-identity-mapping map                           Key-value for the fixed identity mapping which allows to use reserved label for fixed identities (default map[])
+      --fixed-identity-mapping map                           Key-value for the fixed identity mapping which allows to use reserved label for fixed identities
```

Comment:
What do you think of giving an example similar to L20 and L207?

Reply:
good point, let me add example for related field.